htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -1295,7 +1295,9 @@ function setforpredef() {
 		<?php } ?>
 		/* jQuery("#tva_tx, #title_vat").hide(); */
 		/* jQuery("#title_fourn_ref").hide(); */
-		jQuery("#np_marginRate, #np_markRate, .np_marginRate, .np_markRate, #units, #title_units").hide();
+		<?php if (!getDolGlobalString('DISPLAY_MARK_RATES')) { ?>
```

Comment:
There is two vars
DISPLAY_MARGIN_RATES
and
DISPLAY_MARK_RATES

May be you can make a difference with the two ?